pkg/ddc/alluxio/shutdown_test.go

```diff
@@ -1,3 +1,15 @@
+/*
```

Comment:
I suggest you should check the content of license header. thanks.

Reply:
The copyright part is missed and i added them.
Is it right now? Please check it again.